### PR TITLE
[QoL Balance] Shiny Chance from 1/2048 -> 1/1024, shiny charm adjustment

### PR DESCRIPTION
**Pull request by damocleas**
## What are the changes?
Changes the base Shiny Chance after discussion from 1/2048 -> 1/1024

## Why am I doing these changes?
Shinies are a part of the overall progression of the game for most circumstances. 
The *INTENDED* affect is halving the highest end of odds, and making sure the Shiny Charm still ends at 1/32 at 4 stacked

## What did change?
Changed one 32 to a 64 for shiny chance, and a 2 to a 1 for shiny charms as of now.

### Screenshots/Videos
N/A

## How to test the changes?
Merge the PR with an offline version, and/or double check the code to make sure the right lines are adjusted.

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)

### CODE REVIEW

File: src/field/pokemon.ts [hunk 1]
```diff
@@ -1312,7 +1312,7 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
     }
 
     const rand1 = Utils.binToDec(Utils.decToBin(this.id).substring(0, 16));
-    const rand2 = Utils.binToDec(Utils.decToBin(this.id).substring(16, 32));
+    const rand2 = Utils.binToDec(Utils.decToBin(this.id).substring(16, 64));
```
Comment (Tempo-anon) [inline src/field/pokemon.ts:1315]: what are you doing here? You're getting a 48 bit number for rand2

File: src/modifier/modifier.ts [hunk 2]
```diff
@@ -1855,7 +1855,7 @@ export class ShinyRateBoosterModifier extends PersistentModifier {
   }
 
   apply(args: any[]): boolean {
-    (args[0] as Utils.IntegerHolder).value *= Math.pow(2, 2 + this.getStackCount());
+    (args[0] as Utils.IntegerHolder).value *= Math.pow(2, 1 + this.getStackCount());
```
Comment (Tempo-anon) [inline src/modifier/modifier.ts:1858]: The previous shiny charm boosts were 8x, 16x, 32x for 1/2/3. Now the charm boosts are 4x, 8x, 16x. Not sure if you still intend on the first one to be more impactful than the second and third.

**Review by Tempo-anon — CHANGES_REQUESTED** [review]
I literally wrote the documentation on how this code works 😭

**Comment by bennybroseph** [comment]
Why aren't tests passing for this change?

**Pull request closed without merging**